src/View/home.html.twig

```diff
@@ -4,49 +4,53 @@
 
     <!--Thumbnail des catégories -->
     <div class="container">
+        {% set altern = 0 %}
         {% for category, declinations in declinationsByCat %}
             {# add One rectangle by category #}
-            <div class="category-thumbnail">
-                <div class="row">
-                    <div class="hidden-xs {{ (loop.index % 2) ?: 'col-lg-push-8' }} col-lg-3">
-                        <div class="title-block">
-                            <span>{{ category | capitalize }}</span>
+            {% if declinations | length > 1 %}
+                {% set altern = altern + 1 %}
```

Comment:
pour quoi faire ton altern ? tester pair et impair ?
si oui tu peux faire qqchose du genre {% if loop.index0 is odd %}